src/Pages/Home.jsx

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import MainPage from './MainPage.jsx';
```

Comment:
Also move this import to line no 5.
```suggestion
import MainPage from './MainPage';
```